src/main/java/step1/StringCalculator.java

```diff
@@ -0,0 +1,63 @@
+package step1;
+
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    private int result;
+    private static final String SEPERATOR_CHARACTER = ",|:";
+    private static final String CUSTOM_SEPERATOR_CHARACTER = "//(.)\n(.*)";
+    private static final int ZERO = 0;
+
+    public StringCalculator() {
+        this.result = ZERO;
+    }
+
+    public int splitAndSum(String inputString) {
+        if (isNullOrEmpty(inputString)) {
+            return ZERO;
+        }
+        return sum(inputSplit(inputString));
+    }
+
+    private int stringToInteger(String input) {
+        if (input == null || "".equals(input)) {
+            return ZERO;
+        }
+        return Integer.parseInt(input);
+    }
+
+    private int sum(String[] inputSplit) {
+        for (String input : inputSplit) {
+            result += checkNegativeNumber(stringToInteger(input));
+        }
+        return result;
+    }
+
+    private boolean isNullOrEmpty(String input) {
+        if (input == null || "".equals(input)) {
+            return true;
+        }
+        return false;
```

Comment:
아래와 같이 구현할 수 있을 것 같네요 :)
```suggestion
       return input == null || "".equals(input);
```